config/valkyrie.yml

```diff
@@ -3,7 +3,7 @@ development:
   storage_adapter: disk
 
 test:
-  metadata_adapter: memory
+  metadata_adapter: postgres
```

Comment:
Can we make this configurable with env variables?  Or some other way to make it easier to run the tests locally without having to have postgres setup?  Not that it's a huge burden, but it would be nice not to need postgres setup locally to run the tests.